src/config/field-overrides/program.js

```diff
@@ -6,6 +6,10 @@ import PeriodTypeDropDown from '../../forms/form-fields/period-type-drop-down';
  * between these two, as they have different behavior and overrides.
  */
 
+
+// The other options from the backend is not used... yet
+export const featureTypeOverride = ['NONE', 'POINT', 'POLYGON'];
```

Comment:
Do we have the list of all feature types?

It would be easier to maintain a blacklist than a whitelist.